Misc/NEWS.d/next/Core and Builtins/2022-07-28-08-33-31.gh-issue-95355.yN4XVk.rst

```diff
@@ -0,0 +1 @@
+Check tokens[0] after allocating memory
```

Comment:
Thank you for the patch!

By the way, there is no need to go into technical details in a NEWS entry:

```suggestion
``_PyPegen_Parser_New`` now properly detects token memory allocation errors. Patch by Honglin Zhu.
```